src/PrestaShopBundle/Form/Admin/Sell/Order/Delivery/SlipPdfType.php

```diff
@@ -47,20 +47,22 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'date_from',
                 DatePickerType::class,
                 [
-                    'required' => false,
                     'attr' => ['placeholder' => 'YYYY-MM-DD'],
                     'data' => $now,
                     'empty_data' => $now,
+                    'label' => $this->trans('From', 'Admin.Global'),
+                    'help' => $this->trans('Format: 2011-12-31 (inclusive).', 'Admin.Orderscustomers.Help'),
```

Comment:
I know it's out of scope but maybe we can keep a recent date?
```suggestion
                    'help' => $this->trans('Format: %s (inclusive).', 'Admin.Orderscustomers.Help', [date('Y-m-d')]),
```

Reply:
@atomiix I think we have a string translation for `Format: 2011-12-31 (inclusive).`, this is indicative of the requested format, not the current date. (31 makes it obvious that it's the day and not the month)

Reply:
I know, it was just a suggestion to have a exemple with a recent date, I feel like I use a very old software when I see old dates :p

Reply:
Good idea!

Reply:
yep good idea 👍  it should be changed in the translation file then ?